woocommerce/payment-gateway/assets/css/frontend/sv-wc-payment-gateway-payment-form.scss

```diff
@@ -118,19 +118,16 @@ http://www.gnu.org/licenses/gpl-3.0.html
 	}
 }
 
-// add payment method form card images
-#add_payment_method {
-	ul.payment_methods li {
-		list-style-type: none;
+// payment form card images
+ul.payment_methods li {
+
+	div {
+		display: inline-block;
+		margin-left: 22px;
+
 		img {
-			vertical-align: middle;
```

Comment:
seems like we want to apply this to the div, yeah? When loading Add Payment Method (where the form is wide enough for all logos to be inline), they are misaligned with the label: https://cloud.skyver.ge/OAurA22l

Reply:
Good catch! Fixed in 164c505a5cc293b29ee35cea7e78aa4e1c7b98a5.